bootstrap/gentoo.sh

```diff
@@ -1 +0,0 @@
-_gentoo_common.sh
```

Comment:
@cheba Please see the comment from @kuba in the outdated diff above and also my comment mentioning the same like you did. I've split the changes into 2 commits so they can pick what they want. Personally I just care for adding support for Exherbo.